cmd/achcli/describe/file.go

```diff
@@ -88,6 +88,79 @@ func File(ww io.Writer, file *ach.File, opts *Opts) {
 		}
 	}
 
+	// IATBatches
+	for _, iatBatch := range file.IATBatches {
```

Comment:
Can we use `for i := range file.IATBatches`? Those loop assignments are tricky and often include subtle bugs.

Reply:
Sure thing

Reply:
@adamdecaf addressed your comment . please review whenever you have time